apps/meteor/ee/app/livechat-enterprise/server/api/lib/outbound.ts

```diff
@@ -0,0 +1,26 @@
+import type { IOutboundProvider, ValidOutboundProvider } from '@rocket.chat/core-typings';
+import { ValidOutboundProviderList } from '@rocket.chat/core-typings';
+
+import { OutboundMessageProvider } from '../../../../../../server/lib/OutboundMessageProvider';
+
+export class OutboundMessageProviderService {
+	private readonly provider: OutboundMessageProvider;
+
+	constructor() {
+		this.provider = new OutboundMessageProvider();
+	}
+
+	private isProviderValid(type: unknown): type is ValidOutboundProvider {
```

Comment:
```suggestion
	private isProviderValid(type: any): type is ValidOutboundProvider {
```

@MarcosSpessatto , the problem with unknown is that you have to do extra typing shenanigans to make the `includes` work, and at the end, if the value is included we assure it's the one we want, so for this case, i think `any` is fair.